third-party/realdds/include/realdds/dds-device-server.h

```diff
@@ -117,9 +119,9 @@ class dds_device_server
     std::shared_ptr< dds_device_broadcaster > _broadcaster;
     dispatcher _control_dispatcher;
 
-    control_callback _open_streams_callback = nullptr;
     set_option_callback _set_option_callback = nullptr;
     query_option_callback _query_option_callback = nullptr;
+    control_callback _control_callback;
```

Comment:
We set the other callbacks to nullptr and check `if( callback )`. Here you set to empty and check `if( ! callback )`. What is the more efficient way? We should be consistent...

Reply:
The default constructor is "empty" and is the same as "nullptr" and adding `= nullptr` adds exactly nothing and may even cost, just like saying `std::string("")`. The usage is exactly the same as before;...

Reply:
Then lets be consistent, at least in the same file. Add here or remove from the others.

Reply:
Removed.